docker-compose.override.yml.example

```diff
@@ -0,0 +1,13 @@
+x-rails-json-file-logging: &json-file-logging
+  logging:
+    driver: local
+
+services:
+  clock:
+    <<: *json-file-logging
+  initialize_database:
+    <<: *json-file-logging
+  web:
+    <<: *json-file-logging
+  worker:
+    <<: *json-file-logging
```

Comment:
Unfortunately, this list of services that use the `x-rails-config` of `docker-compose.yml` is WET, and I cannot really think of a good / reasonably simple way to avoid that. Not the end of the world, I think.